csrc/validator_utils.h

```diff
@@ -83,7 +83,7 @@ ExpressionEvaluator bindInputsAndLaunchParams(
     const KernelArgumentHolder& aten_inputs,
     const LaunchParams& launch_constraints);
 
-std::vector<std::pair<double, double>> get_val_constants(
+NVF_API std::vector<std::pair<double, double>> getValConstants(
```

Comment:
nitpick: shouldn't this function be named `getValTolerances` instead?

Reply:
It isn't used anywhere else so why not.